.conda-extra-dependencies.yml

```diff
@@ -0,0 +1,3 @@
+- openblas
+- pip
+- spyder-kernels >= 1.10.0
```

Comment:
Looking at this list, I'm not actually convinced we need `openblas` as it should always be installed nowadays anyway I think (and even if conda switches to `accelerate` by default I think that's okay). The `mne-installers` will ensure that `spyder-kernels` can be installed simultaneously for us and they aren't used at all by MNE-Python itself. So maybe the only thing we actually should add is `pip`, in which case maybe it's worth just hard-coding that into the script that generates the `environment.yml` rather than having a separate file?